test/server/spec/bin/builder-core.spec.js

```diff
@@ -21,24 +22,14 @@ var run = require("../../../../bin/builder-core");
 
 var base = require("../base.spec");
 
-// Helpers
-// **Note**: It would be great to just stub stderr, stdout in beforeEach,
-// but then we don't get test output. So, we manually stub with this wrapper.
-var stdioWrap = function (fn) {
-  return function (done) {
-    base.sandbox.stub(process.stdout, "write");
-
-    var _done = function (err) {
-      process.stdout.write.restore();
-      done(err);
-    };
+var readOut = function () {
+  base.mockFs.restore();
+  return fs.readFileSync("stdout.log", { encoding: "utf8" });
```

Comment:
Can you async this?

Side Note: The sync methods when mockFS is active is fine because we're going against in-memory mocks. These appear to be real fs access once you've restored from the mock fs...